pkg/install/v1alpha1/cstor_volume.go

```diff
@@ -528,6 +528,11 @@ spec:
             ports:
             - containerPort: 3260
               protocol: TCP
+            env:
+            - name: QueueDepth
+              value: {{ .Config.QueueDepth.value }}
```

Comment:
I feel this will error out since QueueDepth is not defined as a property of Config

Reply:
How can one determine the best value of QueueDepth in one's environment?
Is there any default value that can be recommended?

Reply:
> How can one determine the best value of QueueDepth in one's environment?
> Is there any default value that can be recommended?

There is default values , but for the tunnings we are going to have a doc/blog with details how someone can determine these values based on there cluster cc @vishnuitta 


> I feel this will error out since QueueDepth is not defined as a property of Config

I got your point to define under DefaultConfig in create-volume CastTemplate but may be its not required as i am setting the default value already here ?

I have tested w/o setting any tunnables in SC and its works .